public/assets/js/task.js

```diff
@@ -1,16 +1,3 @@
-function myFun(val) {
```

Comment:
- Why call it `#sub1`?
- Why hard code routes e.g. `/task/insert`?
- `showMsg(val1, val2)` why name them `val1`, `val2`?
- Why `ajax` error handling also in `success` and not in `error`?